service/prometheus/service.go

```diff
@@ -53,7 +53,8 @@ type Service struct {
 	configMapNamespace string
 	minimumReloadTime  time.Duration
 
-	lastReloadTime time.Time
+	isReloadRequested bool
+	lastReloadTime    time.Time
```

Comment:
Pedantic nitpick, would be nice to order dependencies, internals, settings. At least this is what we do elsewhere.

Reply:
Dependencies, settings, internals is what I've been doing :D 
Let's discuss in #fmt or some such.

Reply:
Questioning state of the art he. Why do you want to make 99% change to your 1%?

Reply:
Do we have a guideline for this already?

Reply:
Yeah, it is written in code every of our projects.

Reply:
Will boyscout separately